### DISCUSSION TITLE
my API stopped responding with an SSL error.  It seems to be on only one of my projects.

### DISCUSSION THREAD

**Discussion by meyerworks** [supabase/supabase#2676] [Questions] [answered]
curl: (35) error:1400410B:SSL routines:CONNECT_CR_SRVR_HELLO:wrong version number

**Comment by inian** [comment, marked as answer]
There is more context for this issue here @meyerworks https://github.com/supabase/supabase/issues/2096#issuecomment-886013053

**Reply by meyerworks** [reply]
Thank you @inian !
